build/generate-docs.ts

```diff
@@ -169,10 +177,17 @@ function convertPropertyToMarkdown(key: string, value: JsonObject, keyPrefix = '
     }
     const valueType = typeToMakrdownLink(value.type);
     if (value.required) {
-        markdown += `Required${valueType}. `;
+        markdown += `Required${valueType}`;
     } else {
-        markdown += `Optional${valueType}. `;
+        markdown += `Optional${valueType}`;
+    }
+
+    if (value.minimum !== undefined || value.maximum !== undefined) {
```

Comment:
Consider moving this if into the format range method so that it will write the entire sentence.

Reply:
I like pure functions better.